src/package/cppwinrt/nuget/Microsoft.Windows.CppWinRT.targets

```diff
@@ -329,7 +329,7 @@ $(XamlMetaDataProviderPch)
         </ItemGroup>
         <PropertyGroup>
           <_MdMergeParameters>-v @(_MdMergeRefsDistinct->'-metadata_dir &quot;%(RelativeDir).&quot;', '&#x0d;&#x0a;')</_MdMergeParameters>
-          <_MdMergeParameters>$(_MdMergeParameters) -o &quot;$(CppWinRTMergedDir)&quot; -i &quot;$(CppWinRTUnmergedDir)&quot; -partial $(_MdMergeDepth)</_MdMergeParameters>
+          <_MdMergeParameters>$(_MdMergeParameters) -o &quot;$(CppWinRTMergedDir.TrimEnd('\'))&quot; -i &quot;$(CppWinRTUnmergedDir.TrimEnd('\'))&quot; -partial $(_MdMergeDepth)</_MdMergeParameters>
```

Comment:
do trailing slashes actually break mdmerge?

Reply:
Yeah it wasn't happy when I tried.